src/status_im/ui/components/list_item/styles.cljs

```diff
@@ -29,6 +29,6 @@
    :height        size})
 
 (def error
-  {:bottom-value 0
+  {:bottom-value -15
```

Comment:
Can you comment or make those defs?

Reply:
@jeluard I'm not sure what to comment here, I tried to make it look exactly the same way is it was before this change, and seems that these magic number comes from other magic number already existing in styles (hundreds of them). So no comments unfortunately.